sdk-python/src/apip_sdk_core/__init__.py

```
@@ -0,0 +1,23 @@
+from importlib.metadata import PackageNotFoundError, version
+
+from . import policy
+from .policy import *  # noqa: F401,F403
+
+try:
+    __version__: str = version("apip-sdk-core")
+except PackageNotFoundError:
+    __version__ = "0.0.0+unknown"
+
+__all__ = list(policy.__all__)
+__all__.extend(["policy", "__version__"])
```

Ruff (0.15.12)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)